### PR TITLE
Create empty TestExecutions in prepareExecutionForStandard

### PR DESCRIPTION
**Pull request by kgevorkyan**
### What's done:
* Provide logic for `prepareExecutionForStandard`: save empty TestExecutions for standard test suites
* Update tests
* Increase block duration in standard test suites scheduler
* Temporary disable kotlin native cache

### CODE REVIEW

File: save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestController.kt [hunk 1]
```diff
@@ -33,6 +33,18 @@ class TestController {
         executionId?.let { testExecutionService.saveTestExecution(executionId, testsIds) }
     }
 
+    /**
+     * @param executionId ID of the [Execution], during which these tests will be executed
+     * @param testSuiteId ID of the [TestSuite], for which there will be created execution in DB
+     */
+    @Suppress("UnsafeCallOnNullableType")
+    @PostMapping("/getAllTestsByTestSuiteIdAndSaveExecution")
+    fun getAllTestsByTestSuiteIdAndSaveExecution(@RequestBody executionId: Long, @RequestParam testSuiteId: Long) {
```
Comment (petertrr) [inline save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestController.kt:42]:
```suggestion
    fun getAllTestsByTestSuiteIdAndSaveExecution(@RequestParam executionId: Long, @RequestParam testSuiteId: Long) {
```
Why not param?

Comment (petertrr) [inline save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestController.kt:42]: Also, why is endpoint called `get...`, if it's `POST` and returns nothing?

File: save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestSuitesService.kt [hunk 2]
```diff
@@ -57,7 +57,7 @@ class TestSuitesService {
      * @return all standard test suites with specific name
      */
     fun findStandardTestSuitesByName(name: String) =
-            testSuiteRepository.findAllByNameAndType(name, TestSuiteType.STANDARD).map { it.toDto() }
+            testSuiteRepository.findAllByNameAndType(name, TestSuiteType.STANDARD)
```
Comment (petertrr) [inline save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestSuitesService.kt:60]: Why did you change return type?

File: save-backend/src/main/kotlin/org/cqfn/save/backend/repository/TestRepository.kt [hunk 3]
```diff
@@ -12,4 +12,6 @@ import java.util.Optional
 @Suppress("MISSING_KDOC_CLASS_ELEMENTS", "MISSING_KDOC_ON_FUNCTION")
 interface TestRepository : BaseEntityRepository<Test> {
     fun findByHashAndFilePathAndTestSuiteId(hash: String, filePath: String, testSuiteId: Long): Optional<Test>
+
+    fun findAllByTestSuiteIdIs(testSuiteId: Long): List<Test>
```
[the remark this answers is not in the document]
Comment (kgevorkyan) [reply save-backend/src/main/kotlin/org/cqfn/save/backend/repository/TestRepository.kt:16]: Fixed

File: save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestController.kt (hunk 1 above)
Comment (kgevorkyan) [reply save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestController.kt]: Because I'd like to pass to the uri only the `testSuiteId` and `executionId` by `BodyInserters` as more low level and static information

Comment (kgevorkyan) [reply save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestController.kt]: Because it also get tests from DB by `testSuiteId`, however it could be confusing, renamed

File: save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestSuitesService.kt (hunk 2 above)
Comment (kgevorkyan) [reply save-backend/src/main/kotlin/org/cqfn/save/backend/service/TestSuitesService.kt:60]: For now I use this method also from preprocessor, but `dto` is not enough for him, since we need the `id`, which located only in `TesSuite`. So I change the return type and what about `orchestrator` - he called `it.toDto()` by itself

**Review by petertrr — APPROVED** [review]
